taqueria-plugin-flextesa/index.ts

```diff
@@ -49,6 +49,57 @@ Plugin.create(_i18n => ({
             ],
             encoding: "json"
         }),
+        Task.create({
+            task: "typecheck",
+            command: "typecheck <sandboxName> [sourceFiles...]",
+            aliases: ["tc"],
+            description: "Typecheck Michelson contracts",
+            options: [],
+            handler: "proxy",
+            positionals: [
+                PositionalArg.create({
+                    placeholder: "sandboxName",
+                    description: "The name of the sandbox to use"
+                }),
+                PositionalArg.create({
+                    placeholder: "sourceFiles",
+                    description: "The names of the Michelson contracts you wish to typecheck, separated by space"
+                }),
+            ],
+            encoding: "json"
+        }),
+        Task.create({
+            task: "simulate",
+            command: "simulate <sandboxName> <sourceFile>",
+            aliases: ["sim"],
+            description: "Run Michelson contracts as a simulation",
+            options: [
+                Option.create({
```

Comment:
Can we be more explicit about the expected input format for both options? (ie string? JSON? quotes? etc)

Reply:
Ok will reflect in the next commit